### ISSUE TITLE
jsx-handler-names errors for Ref handlers

### ISSUE THREAD

**Issue by AndyOGo** [yannickcr/eslint-plugin-react#2406]
I expect handler functions for React refs not to fail, and related props should not be `on-` prefixed.

```sh
 135:35  error  Prop key for handleStrokeRef must begin with 'on'                              react/jsx-handler-names
```

```js
<StepperStroke strokeRef={this.handleStrokeRef} key='stepper-stroke' />}
```

**Comment by ljharb** [comment]
Why would a ref callback be named “handle”?

**Comment by AndyOGo** [comment]
Good point, I will use set prefix
